test/cookbooks/sensu_test/recipes/default.rb

```diff
@@ -5,3 +5,9 @@
 sensu_ctl 'default' do
   action [:install, :configure]
 end
+
+sensu_check 'cron' do
```

Comment:
we should not assume cron is default and should specify it here.

Reply:
added